[Features] Configuration

DreaMinder opened the pull request:
Сама страница: https://ru.nuxtjs.org/docs/2.x/features/configuration

DreaMinder committed:
add initial eng page

AndreiDetenkov commented:
взял в работу

AndreiDetenkov committed:
translated configuration.md

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -1,107 +1,101 @@
 ---
 title: Configuration
-description: By default, Nuxt.js is configured to cover most use cases. This default configuration can be overwritten with the nuxt.config.js file.
+description: По умолчанию Nuxt.js настроен так, чтобы охватить большую часть настроек по умолчанию. Эта конфигурация может быть перезаписана в файле nuxt.config.js.
```
```suggestion
description: Nuxt.js настроен так, чтобы охватить большую часть настроек по умолчанию. Эта конфигурация может быть перезаписана в файле nuxt.config.js.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -1,107 +1,101 @@
+      - да
+      - нет
+    correctAnswer: нет
+  - question: Какой хост будет присвоен по-умолчанию в режиме разработки Nuxt.js?
```
```suggestion
  - question: Какой хост будет присвоен по умолчанию в режиме разработки Nuxt.js?
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -1,107 +1,101 @@
       - style="scss"
     correctAnswer: lang="scss"
-  - question: What is the default Nuxt.js development server port?
+  - question: Какой порт булет присвоен по-умолчанию в режиме разработки Nuxt.js?
```
```suggestion
  - question: Какой порт будет присвоен по умолчанию в режиме разработки Nuxt.js?
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -1,107 +1,101 @@
+      - Да
+      - Нет
+    correctAnswer: Да
+  - question: Какое ствойство вы будете использовать в файле nuxt.config.js для расширения конфигурации webpack?
```
```suggestion
  - question: Какое свойство вы будете использовать в файле nuxt.config.js для расширения конфигурации webpack?
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -1,107 +1,101 @@
       - globalCss
     correctAnswer: css
-  - question: You can use JSX in your Nuxt.js application?
+  - question: Можете ли вы испольщовать синтаксис JSX в вашем Nuxt.js приложении?
```
```suggestion
  - question: Можете ли вы использовать синтаксис JSX в вашем Nuxt.js приложении?
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -1,107 +1,101 @@
       - localhost
     correctAnswer: 3000
-  - question: In the nuxt.config.js what property do you use to add global CSS files?
+  - question: Какое ствойство вы будете использовать в файле nuxt.config.js для добавления глобальных CSS файлов?
```
```suggestion
  - question: Какое свойство вы будете использовать в файле nuxt.config.js для добавления глобальных CSS файлов?
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -1,107 +1,101 @@
       - .ignorenuxt
     correctAnswer: .nuxtignore
-  - question: If you want to ignore the about file of your Nuxt.js app what prefix would you use?
+  - question: Если вы хотите игнорировать все файлы с названием about в вашем Nuxt.js прилодении, какой перфикс вы будете использовать?
```
```suggestion
  - question: Если вы хотите игнорировать все файлы с названием about в вашем Nuxt.js приложении, какой перфикс вы будете использовать?
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -1,107 +1,101 @@
 ---
 
-By default, Nuxt.js is configured to cover most use cases. This default configuration can be overwritten with the nuxt.config.js file.
+По умолчанию Nuxt.js настроен так, чтобы охватить большую часть настроек по умолчанию. Эта конфигурация может быть перезаписана в файле nuxt.config.js.
```
```suggestion
Nuxt.js настроен так, чтобы охватить большую часть настроек по умолчанию. Эта конфигурация может быть перезаписана в файле nuxt.config.js.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -1,107 +1,101 @@
 </base-alert>
 
-In `nuxt.config.js`, add the CSS resources:
+В файле `nuxt.config.js` добавьте пути до css файлов:
```
```suggestion
В файле `nuxt.config.js` добавьте пути до CSS файлов:
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -111,17 +105,17 @@ export default {
+Спасибо [загрузчику Vue](http://vue-loader.vuejs.org/en/configurations/pre-processors.html), вы можете использовать самые разные препроцессоры в `<template>` или `<style>`: используя `lang` атрибут.
 
-Example of our `pages/index.vue` using [Pug](https://github.com/pugjs/pug) and [Sass](http://sass-lang.com/):
+Это пример (`pages/index.vue` удалить из контекста?) использования препроцессоров [Pug](https://github.com/pugjs/pug) или [Sass](http://sass-lang.com/):
```
```suggestion
Это пример использования препроцессоров [Pug](https://github.com/pugjs/pug) или [Sass](http://sass-lang.com/):
```
ok 👍

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -156,9 +150,9 @@ npm install --save-dev sass sass-loader fibers
 ## JSX
 
-Nuxt.js uses [@nuxt/babel-preset-app](https://github.com/nuxt/nuxt.js/tree/dev/packages/babel-preset-app), which is based on the official [@vue/babel-preset-app](https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/babel-preset-app) for babel default configuration, so you can use JSX in your components.
+Nuxt.js использует пресет [@nuxt/babel-preset-app](https://github.com/nuxt/nuxt.js/tree/dev/packages/babel-preset-app), который основан на официальном пресете [@vue/babel-preset-app](https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/babel-preset-app) для концигурации babel по-умолчанию, так что вы можете использовать синтаксис JSX в ваших компонентах.
```
```suggestion
Nuxt.js использует пресет [@nuxt/babel-preset-app](https://github.com/nuxt/nuxt.js/tree/dev/packages/babel-preset-app), который основан на официальном пресете [@vue/babel-preset-app](https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/babel-preset-app) для конфигурации babel по умолчанию, что позволяет использовать синтаксис JSX в ваших компонентах.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -156,9 +150,9 @@ npm install --save-dev sass sass-loader fibers
+Nuxt.js использует пресет [@nuxt/babel-preset-app](https://github.com/nuxt/nuxt.js/tree/dev/packages/babel-preset-app), который основан на официальном пресете [@vue/babel-preset-app](https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/babel-preset-app) для концигурации babel по-умолчанию, так что вы можете использовать синтаксис JSX в ваших компонентах.
 
-You can also use JSX in the `render` method of your components:
+Так же вы можете использовать синтаксис JSX в методе `render` вашего компонента:
```
```suggestion
Также вы можете использовать синтаксис JSX в методе `render` вашего компонента:
```

AndreiDetenkov replied:
тут я немного не согласен. [погуглив](https://russkiiyazyk.ru/orfografiya/pravopisanie/takzhe-i-tak-zhe-slitno-ili-razdelno.html) я все таки склоняюсь к тому что написание должно быть раздельным.

DreaMinder replied:
хм, чекнул ссылку, но все же:
Также - это как написать "кроме того", как было в оригинале.
Так же - это как написать "таким же способом".
Но ок, пусть будет.

DreaMinder commented on content/ru/guides/features/configuration.md:
````diff
@@ -173,55 +167,55 @@ export default {
 ```
 
-Aliasing `createElement` to `h` is a common convention you’ll see in the Vue ecosystem but is actually optional for JSX since it [automatically injects](https://github.com/vuejs/babel-plugin-transform-vue-jsx#h-auto-injection) `const h = this.$createElement` in any method and getter (not functions or arrow functions) declared in ES2015 syntax that has JSX so you can drop the (h) parameter.
+Aliasing (?) `createElement` в `h` это общая конвенция, которую вы можете увидеть в экосистеме Vue, но не являющаяся обязательной для синтаксиса JSX, с тех пор как это [автоматически вставляется](https://github.com/vuejs/babel-plugin-transform-vue-jsx#h-auto-injection) `const h = this.$createElement` в любом методе или геттере (без использования обычных или стрелочных функции) объявленом в синтас ...
````
```suggestion
Присваивание псевдонима `h` для метода `createElement` - это общая конвенция, которую можно встретить в экосистеме Vue. Но она необязательна для синтаксиса JSX, так как `const h = this.$createElement` [автоматически вставляется](https://github.com/vuejs/babel-plugin-transform-vue-jsx#h-auto-injection) в любом методе или геттере (без использования обычных или стрелочных функций) объявленном в синтаксисе ES2015 где есть JSX, так что вы можете пропустить параметр `h`.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -173,55 +167,55 @@ export default {
+### Свойство ignorePrefix
 
-Any file in pages/, layout/, middleware/ or store/ will be ignored during the build if its filename starts with the prefix specified by ignorePrefix.
+Некоторые файлы в папках pages/, layout/, middleware/ или store/ будет игнорироваться при сборке проекта, если названия файлов начинаются с специального префикса ignorePrefix.
```
```suggestion
Некоторые файлы в папках pages/, layout/, middleware/ или store/ будет игнорироваться при сборке проекта, если названия файлов начинаются со специального префикса ignorePrefix.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -173,55 +167,55 @@ export default {
+Некоторые файлы в папках pages/, layout/, middleware/ или store/ будет игнорироваться при сборке проекта, если названия файлов начинаются с специального префикса ignorePrefix.
 
-By default all files which start with `-` will be ignored, such as `store/-foo.js` and `pages/-bar.vue`. This allows for co-locating tests, utilities, and components with their callers without themselves being converted into routes, stores, etc.
+По умолчанию, все файлы начинающиеся с `-` будут игнорироваться, например `store/-foo.js` или `pages/-bar.vue`. This allows for co-locating tests, utilities, and components with their callers without themselves being converted into routes, stores, etc. ? хз:)
```
```suggestion
По умолчанию, все файлы начинающиеся с `-` будут игнорироваться, например `store/-foo.js` или `pages/-bar.vue`. This allows for co-locating tests, utilities, and components with their callers without themselves being converted into routes, stores, etc. ? хз:)
```
```suggestion
По умолчанию, все файлы начинающиеся с `-` будут игнорироваться, как например `store/-foo.js` или `pages/-bar.vue`. Это позволяет размещать рядом файлы тестов, утилит и компонентов избегая автоматической конвертации их в компоненты роутера или хранилища.
```

DreaMinder replied:
хз почему здесь 2 suggestions, первый можно игнорить

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -231,7 +225,7 @@ export default {
 ### ignoreOptions
 
-`nuxtignore` is using `node-ignore` under the hood, `ignoreOptions` can be configured as `options` of `node-ignore`.
+`nuxtignore` использует `node-ignore` под капотом, поэтому `ignoreOptions` могут быть настроены как `options` of `node-ignore`.
```
```suggestion
`nuxtignore` использует пакет `node-ignore` под капотом, поэтому `ignoreOptions` могут быть настроены как свойство `options` пакета `node-ignore`.
```

AndreiDetenkov replied:
`nuxtignore` использует пакет `node-ignore` под капотом, поэтому `ignoreOptions` могут быть настроены как свойство `options` пакета `node-ignore`.

DreaMinder replied:
ок, пойдет

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -241,9 +235,9 @@ export default {
+## Расширяемая конфигурация webpack
 
-You can extend nuxt's webpack configuration via the `extend` option in your `nuxt.config.js`. The `extend` option of the `build` property is a method that accepts two arguments. The first argument is the webpack `config` object exported from nuxt's webpack config. The second parameter is a context object with the following boolean properties: `{ isDev, isClient, isServer, loaders }`.
+Вы можете расширить конфигурацию webpack в nuxt.js с помощью метода `extend` в вашем конфигурационном файле `nuxt.config.js`. Метод `extend` является свойством объекта `build` и принимает два аргумента. Первый аргумент - это объект конфигураций `config`, экспортируемый из nuxt's webpack config. Второй аргумент - это context объект включающий в себя такие булевые свойства как: `{ isDev, isClient,  ...
```
```suggestion
Вы можете расширить конфигурацию webpack в nuxt.js с помощью метода `extend` в вашем конфигурационном файле `nuxt.config.js`. Метод `extend` является свойством объекта `build` и принимает два аргумента. Первый аргумент - это объект конфигураций `config`, экспортируемый из конфигурации webpack в фреймворке Nuxt. Второй аргумент - это context объект включающий в себя такие булевые свойства как: `{ isDev, isClient, isServer, loaders }`.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -263,11 +257,11 @@ export default {
+### Настройка конфигурации чанков (chunks)
 
-You may want to tweak the [optimization configuration](/docs/2.x/configuration-glossary/configuration-build#optimization) a bit, avoiding a rewrite of the default object.
+Возможно, вы захотите немного подкорректировать [конфигурацию оптимизации](/docs/2.x/configuration-glossary/configuration-build#optimization), избегая перезаписи объекта по умолчанию.
```
```suggestion
Возможно, вы захотите немного подкорректировать [настройки оптимизации](/docs/2.x/configuration-glossary/configuration-build#optimization), избегая перезаписи объекта стандартных настроек.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -281,9 +275,9 @@ export default {
+### Проверка линтером ESLint при каждой сборке webpack в режиме разработки
 
-In order to be aware of code style errors, you may want to run [ESLint](https://github.com/webpack-contrib/eslint-loader) on every build in the dev environment.
+Чтобы быть в курсе ошибок в проекте, возможно вы захотите запустить [ESLint](https://github.com/webpack-contrib/eslint-loader) при каждой сборке проета в режиме разработки.
```
```suggestion
Чтобы быть в курсе ошибок в проекте, следует запускать [ESLint](https://github.com/webpack-contrib/eslint-loader) при каждой сборке проета в режиме разработки.
```
you may want to - речевой оборот, думаю не стоит переводить буквально

AndreiDetenkov replied:
ok

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -302,57 +296,56 @@ export default {
+### Проверка конфигурации webpack
 
-## Edit host and port
+Для сложных проектов и отладки иногда полезно проверить, как будет выглядеть окончательная настройка конфигурации webpack. К счастью, вы можете запустить команду `nuxt webpack` мз вашего проекта для вывода конфигурации. Проверьте этот пулл-реквест [#7029](https://github.com/nuxt/nuxt.js/pull/7029) для получения более детальной информации.
```
```suggestion
Для сложных проектов и отладки иногда полезно проверить, как будет выглядеть окончательная настройка webpack. К счастью, вы можете запустить команду `nuxt webpack` из вашего проекта для вывода конфигурации. Взгляните на этот пулл-реквест [#7029](https://github.com/nuxt/nuxt.js/pull/7029) для более детальной информации.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -302,57 +296,56 @@ export default {
 
-Host `'0.0.0.0'` is designated to tell Nuxt.js to resolve a host address, which is accessible to connections _outside_ of the host machine (e.g. LAN). If the host is assigned the string value of `'0'` (not 0, which is falsy), or `'0.0.0.0'` your local IP address will be assigned to your Nuxt.js application.
+По умолчанию хостом сервера разработки Nuxt.js является `localhost`, который доступен только внутри хост-машины. Для просмотра приложения на другом устройстве необходимо изменить хост. Вы можете изменить его в файле nuxt.config.js.
+Host `'0.0.0.0'` is designated to tell Nuxt.js to resolve a host address, which is accessible to connections _outside_ of the host machine (e.g. LAN). Хост `'0.0.0.0'` предназначен для того, чтобы Nuxt.js мог испольщовать хост адрес, который доступен подключениям _снаружи_ хост-машины (например, LAN). Если хосту присвоено строковое значение `'0'` (не 0, что является falsy), или `'0.0.0.0'0'`, то  ...
```
```suggestion
Хост `'0.0.0.0'` предназначен для того, чтобы Nuxt.js мог использовать хост адрес, который доступен подключениям _снаружи_ хост-машины (например, LAN). Если хосту присвоено строковое значение `'0'` (а не число 0, которое приводится к false), или `'0.0.0.0'0'`, то ваш локальный IP-адрес будет присвоен вашему Nuxt.js приложению.
```

AndreiDetenkov replied:
(а не число 0, которое приводится к false)

DreaMinder replied:
круто, согласен

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -302,57 +296,56 @@ export default {
 export default {
   server: {
-    host: '0' // default: localhost
+    host: '0' // по-умолчанию: localhost
```
```suggestion
    host: '0' // по умолчанию: localhost
```

DreaMinder commented on content/ru/guides/features/configuration.md:
````diff
@@ -302,57 +296,56 @@ export default {
 ```
 
-You can also change the port number from the default port of 3000.
+Также вы можете изменить номер вашего порта, по-умолчанию который равен 3000.
````
```suggestion
Также вы можете изменить номер вашего порта, который по умолчанию равен 3000.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -302,57 +296,56 @@ export default {
 export default {
   server: {
-    port: 8000 // default: 3000
+    port: 8000 // по-умолчанию: 3000
```
```suggestion
    port: 8000 // по умолчанию: 3000
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -302,57 +296,56 @@ export default {
 <base-alert type="info">
 
-If the port is assigned the string value of `'0'` (not 0, which is falsy) a random port number will be assigned to your Nuxt.js application.
+Если порту присвоено строковое значение `'0'` (не 0, что является falsy), вашему Nuxt.js приложению будет присвоен случайный номер порта.
```
```suggestion
Если порту присвоено строковое значение `'0'` (а не число 0, которое приводится к false), вашему Nuxt.js приложению будет присвоен случайный номер порта.
```

AndreiDetenkov replied:
"(когда 0 не является числом)" может так?

AndreiDetenkov replied:
решил оставить (а не число 0, которое приводится к false)

DreaMinder replied:
поддержу

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -302,57 +296,56 @@ export default {
 </base-alert>
 
-Although you can modify this in the nuxt.config.js file it is not advised to as it might cause you issues when hosting your site. It is much better to modify the host and port direct in the dev command.
+Хотя вы можете изменить это в файле nuxt.config.js, это не рекомендуется, так как это может вызвать проблемы при хостинге вашего сайта. Гораздо лучше изменить хост и порт непосредственно в команде dev.
```
```suggestion
Хотя вы можете изменить эти настройки в файле nuxt.config.js, это не рекомендуется, так как это может вызвать проблемы при загрузке вашего сайта на хостинг. Гораздо лучше изменить хост и порт непосредственно в команде dev.
```

AndreiDetenkov replied:
... при загрузке вашего сайта на хостинг. 
все таки по русски мы загружаем данные на сервер, хотя по английски это будет upload

DreaMinder replied:
ок, давай будет "загрузка"

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -302,57 +296,56 @@ export default {
+## Асинхронная конфигурация
 
-Although it is better to use the normal configuration `export default {}` you can have an async configuration by exporting an async function that return the config object.
+К тому же лучше использовать обычную конфигурацию `export default {}`, тогда вы можете получить асинхронную конфигурацию, экспортировав функцию async, которая возвращает объект.
```
```suggestion
Вы можете использовать асинхронную конфигурацию, экспортировав функцию с ключевым словом async, которая возвращает объект. Хотя в большинстве случаев лучше использовать обычную конфигурацию `export default {}`.
```

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -370,15 +363,15 @@ export default async () => {
 <base-alert>
 
-The axios-module cannot be used in `nuxt.config.js`. You will need to import axios and configure it again.
+Модуль axios-module не может быть использован в файле `nuxt.config.js`. Вам нужно будет импортировать модуль axios и настроить его заново.
```
```suggestion
Модуль nuxt-axios не может быть использован в файле `nuxt.config.js`. Вам нужно будет импортировать пакет axios и настроить его с нуля.
```

AndreiDetenkov replied:
о каком модуле идет речь? о библиотеке axios? просто модуль @nuxt/axios как раз таки импортируется в nuxt.config.js а если мы говорим о axios тогда можно вставить ссылку на аксиос либу? и не писать что это модуль?

DreaMinder replied:
Модуль - @nuxt/axios
Пакет, библиотека - axios

тут имеется ввиду что в нухт конфиге модулем запросы делать не выйдет и нужно делать:
import axios from 'axios' вместо этого
там не импортируется
import axios from '@nuxt/axios'
а модуль просто подключается, это ненмого другое...

так что думаю мое предложение выглядит адекватно

DreaMinder commented on content/ru/guides/features/configuration.md:
```diff
@@ -370,15 +363,15 @@ export default async () => {
 <base-alert type="next">
 
-The `nuxt.config.js` has way more customization and configuration options! Check out all its keys in the [configuration glossary](/docs/2.x/configuration-glossary/configuration-build).
+TВ `nuxt.config.js` гораздо больше возможностей настройки и конфигурирования! Проверить все возможности можно в [глосарий конфигураций](/docs/2.x/configuration-glossary/configuration-build).
```
```suggestion
В `nuxt.config.js` гораздо больше возможностей настройки и конфигурирования! Проверить все возможности можно в [глоссарии конфигураций](/docs/2.x/configuration-glossary/configuration-build).
```

DreaMinder reviewed (commented):
Oof, I almost had a stroke while trying to translate some parts. At least it's done now, almost there

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

действительно без дефиса )

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

согласен

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

согласен. я в этом месте сломался немного.

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

согласен. нашел правило (https://russkiypro.ru/pravilnoe-pravopisanie-s-ili-so/)

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

AndreiDetenkov committed:
Update content/ru/guides/features/configuration.md

Co-authored-by: DreaMinder <dreaminder7@gmail.com>

DreaMinder commented:
Все круто, спасибо, мерджу
